modelon/impact/client/experiment_definition.py

```diff
@@ -206,6 +216,87 @@ def with_modifiers(self, modifiers=None, **modifiers_kwargs):
             )
         return new
 
+    def with_extensions(self, *experiment_extension):
+        """Sets up an experiment definitions with multiple experiment extensions.
+
+        Parameters:
+
+            experiment_extension --
+                "A variable-length argument of experiment extensions."
+                "Extension object must either be a dictionary or an instance of "
```

Comment:
I don't think we should allow dictionary as input for this case. The extra complexity is not worth it and it just gives the user more ropes to tangle themself into.